op-node/rollup/derive/channel_test.go

```diff
@@ -145,12 +145,20 @@ func TestFrameValidity(t *testing.T) {
 
 func TestBatchReader(t *testing.T) {
 	rng := rand.New(rand.NewSource(0x543331))
-	singularBatch := RandomSingularBatch(rng, 20, big.NewInt(333))
-	batchDataInput := NewBatchData(singularBatch)
+
+	batchCount := 2
```

Comment:
```suggestion
	batchCount := 3
```
Let's do 3 now that we use slices.

Reply:
Done in c21424f09f.